app/src/main/java/de/westnordost/streetcomplete/overlays/cycleway/SeparateCyclewayForm.kt

```diff
@@ -21,7 +21,7 @@ import org.koin.android.ext.android.inject
 class SeparateCyclewayForm : AImageSelectOverlayForm<SeparateCycleway>() {
 
     override val items: List<DisplayItem<SeparateCycleway>> get() =
-        listOf(PATH, NON_DESIGNATED_ON_FOOTWAY, NON_SEGREGATED, SEGREGATED, EXCLUSIVE_WITH_SIDEWALK, EXCLUSIVE).map {
+        listOf(PATH, NON_DESIGNATED_ON_FOOTWAY, ALLOWED_ON_FOOTWAY, NOT_ALLOWED, NON_SEGREGATED, SEGREGATED, EXCLUSIVE_WITH_SIDEWALK, EXCLUSIVE).map {
```

Comment:
Now, this should be possible.

```suggestion
        SeparateCycleway.entries.map {
```